docs/components/query.md

```diff
@@ -274,7 +274,7 @@ Flags:
                                  in all alerts 'Source' field.
       --enable-feature= ...      Comma separated experimental feature names
                                  to enable.The current list of features is
-                                 query-pushdown.
+                                 query-pushdown,promql-at-modifier,promql-negative-offset.
```

Comment:
The docs are auto-generated from the flag descriptions. You can update the values here https://github.com/thanos-io/thanos/blob/main/cmd/thanos/query.go#L188 and run `make docs` to fix the CI.